scipy/stats/tests/test_discrete_basic.py

```diff
@@ -190,8 +190,8 @@ def check_cdf_ppf(distfn, arg, supp, msg):
     cdf_supp = distfn.cdf(supp, *arg)
     # In very rare cases, the finite precision calculation of ppf(cdf(supp))
     # can produce an array in which an element is off by one.  We nudge the
-    # CDF values down by 10 ULPs help to avoid this.
-    cdf_supp0 = cdf_supp - 10*np.spacing(cdf_supp)
+    # CDF values down by 15 ULPs help to avoid this.
+    cdf_supp0 = cdf_supp - 15*np.spacing(cdf_supp)
```

Comment:
We can try this to get this PR merged, but preferably the PPF would not step to the next integer at a value this much lower than the true threshold. It would be good to track down the root cause.

Reply:
I agree; I actually didn't think that this change was going to work, but surprised it did.  @mborland said he would take a closer look tomorrow, but this at least hints at what's going on -- some loss of precision under the hood.  Could be due to our no long double promotion policy?

Reply:
Under investigation in https://github.com/boostorg/math/issues/935